docs/configure-kibana-endpoint.asciidoc

```diff
@@ -1,12 +1,15 @@
 [[setup-kibana-endpoint]]
 == Configure the Kibana endpoint
 
-Starting with {beatname_uc} 7.3.0, we've introduced support for
-{kib-ref}/agent-configuration.html[APM Agent configuration in Kibana] via the Kibana API.
-This requires a Kibana endpoint configuration.
+Starting with {beatname_uc} 7.3.0, we've introduced a new Kibana API for APM Server.
+Configuring this endpoint is required for 
+{kib-ref}/agent-configuration.html[APM Agent configuration in Kibana].
 Once the endpoint is configured,
 you'll be able to fine-tune certain agent configurations directly in Kibana without having to redeploy your agents.
 
+IMPORTANT: Kibana must be running and healthy before starting APM Server.
```

Comment:
After my suggestion, we discussed and prioritized https://github.com/elastic/apm-server/issues/2371. Keep an eye on it, hopefully this will have to be removed again (sorry for the back and forth)

Reply:
Not a problem at all 😃